server/src/db/seeders/SquidSeeder.js

```diff
@@ -7,7 +7,8 @@ export class SquidSeeder {
     console.log("Seeding squids...");
 
     try {
-      await new Factory(Squid).createMany(3);
```

Comment:
@CassandraNewell thoughts on having a delete at the top to allow for re-seeding?

Reply:
I'm fine with that! 🙌🏻